packages/rea/rea.1.0.0/opam

```diff
@@ -0,0 +1,23 @@
+opam-version: "2.0"
+synopsis: "Effectful OCaml with Objects and Variants"
+maintainer: "Vesa Karvonen <vesa.a.j.k@gmail.com>"
+authors: "Vesa Karvonen"
+license: "MIT"
+homepage: "https://github.com/polytypic/rea-ml"
+bug-reports: "https://github.com/polytypic/rea-ml/issues"
+dev-repo: "git+https://github.com/polytypic/rea-ml.git"
+depends: [
+  "dune" {>= "3.3.0"}
+  "ocaml" {>= "4.08.0"}
+  "ppx_optcomp" {>= "v0.14.0"}
+  "ocamlformat" {= "0.23.0" & with-test}
+  "odoc" {>= "2.1.1" & with-doc}
```

Comment:
Your package definition does not have tests and does not compile the documentation so these definitions are superfluous 
```suggestion
```